src/main/kotlin/org/jetbrains/research/commentupdater/inspection/CodeCommentInspection.kt

```diff
@@ -0,0 +1,116 @@
+package org.jetbrains.research.commentupdater.inspection
+
+import com.intellij.codeInspection.*
+import com.intellij.openapi.vcs.changes.Change
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.vcs.changes.ChangeListManager
+
+import com.intellij.psi.*
+import com.intellij.psi.javadoc.PsiDocComment
+import gr.uom.java.xmi.diff.*
+import org.jetbrains.research.commentupdater.CodeCommentDiffs
+import org.jetbrains.research.commentupdater.JITDetector
+import org.jetbrains.research.commentupdater.processors.CodeCommentTokenizer
+import org.jetbrains.research.commentupdater.processors.MethodChangesExtractor
+import org.jetbrains.research.commentupdater.processors.RefactoringExtractor
+import org.refactoringminer.api.Refactoring
+import org.refactoringminer.api.RefactoringType
+import kotlin.io.path.ExperimentalPathApi
+
+
+class CodeCommentInspection : AbstractBaseJavaLocalInspectionTool() {
+
+    private val LOG: Logger = Logger.getInstance("#org.jetbrains.research.commentupdater.inspection.CodeCommentInspection")
+    val detector = JITDetector()
+
+    var currentFile = ""
+    var currentChanges: Change? = null
+    var currentMethodsRefactorings = hashMapOf<String, MutableList<Refactoring>>()
+
+    val DESCRIPTION_TEMPLATE = "Inconsistent comment found"
+
+    override fun inspectionStarted(session: LocalInspectionToolSession, isOnTheFly: Boolean) {
+        LOG.info("Inspection started")
```

Comment:
Let's add the plugin's name at the beginning of each log message (`"[CommentUpdater] text"`), it would help with identifying the plugin-related logs. 
What do you think?

Reply:
I used `javaClass` as a parameter for `Logger.getInstance` to identify plugin's logs, but I think your solution is better. I added this tag to all logs.